src/pages/KeywordPage.jsx

```diff
@@ -22,20 +28,26 @@ export default function KeywordPage() {
     );
   else
     return (
-      <section className="mx-auto box-border flex w-full p-10">
-        <div className="flex flex-wrap justify-center">
-          {searchResult.map((product) => (
-            <Card
-              key={product.product_code}
-              name={product.name}
-              price={product.price}
-              imageUrl={product.image_url}
-              onClickEvent={() => {
-                search(product.image_url);
-              }}
-            />
-          ))}
-        </div>
-      </section>
+      <div>
+        {loading ? (
+          <Loading />
+        ) : (
+          <div className="bg-white">
+            <div className="mx-auto max-w-2xl py-16 px-4 sm:py-24 sm:px-6 lg:max-w-7xl lg:px-8">
+              <div className="mt-6 grid grid-cols-1 gap-y-10 gap-x-6 sm:grid-cols-2 lg:grid-cols-4 xl:gap-x-8">
+                {searchResult.map((product) => (
+                  <div key={product.product_code} className="group relative">
+                    <Card
+                      name={product.name}
+                      price={product.price}
+                      imageUrl={product.image_url}
+                    />
+                  </div>
+                ))}
+              </div>
+            </div>
+          </div>
+        )}
+      </div>
```

Comment:
```suggestion
      <section className="mx-auto box-border flex w-full p-10">
        <div className="flex flex-wrap justify-center">
          {searchResult.map((product) => (
            <Card
              key={product.product_code}
              name={product.name}
              price={product.price}
              imageUrl={product.image_url}
              onClickEvent={() => {
                search(product.image_url);
              }}
            />
          ))}
        </div>
      </section>
```
빨간색 부분을 초록색 부분으로 수정해주세요!

Reply:
Commit suggestion 버튼을 누르시면 됩니다